ansible/vagrant/vagrant-ansible.yml

```diff
@@ -1,19 +1,20 @@
 - hosts: all
   sudo: yes
+  vars:
+    - public_iface: eth0
```

Comment:
@eparis Are you sure? I tested that by default `eth0` is used in my environment, but I'm not sure is Ansible cached some facts from previous runs (I don't explicitly use any facts caching, but not sure is Ansible does any).

In this part Ansible *will* gather facts, but I'm unsure whether this will be done before evaluating `vars:` part or after:

```
- hosts: all
  sudo: yes
  vars:
    - public_iface: "{{ ansible_default_ipv4.interface }}"
  tasks:
    - ...
```

If my tests are correct, it gather facts before evaluating `vars:` (at least in my version of Ansible), and these changes works correctly.